net-misc/ubridge/ubridge-0.9.14.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit fcaps user
+
+DESCRIPTION="Bridge for UDP tunnels, Ethernet, TAP and VMnet interfaces."
+HOMEPAGE="https://github.com/GNS3/ubridge"
+SRC_URI="https://github.com/GNS3/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+system-iniparser"
+
+RDEPEND="
+	net-libs/libpcap
+	system-iniparser? ( dev-libs/iniparser:* )
+"
+DEPEND="
+	${RDEPEND}
+"
+
+PATCHES=( "${FILESDIR}"/${P}-respect-flags.patch )
+
+pkg_setup() {
+	enewgroup ubridge
+}
+
+src_configure() {
+	if use system-iniparser; then
+		export SYSTEM_INIPARSER=1
+	else
+		unset SYSTEM_INIPARSER
+	fi
+}
+
+src_install() {
+	insinto /usr/bin
```

Comment:
Please use `exeinto`/`exeopts`/`doexe` for executables. I know that you override all the stuff anyway but if someone adds non-executable files in the future, it would be cleaner.